### DISCUSSION TITLE
WEBINAR: Using OpenDDS’s RtpsRelay to Connect IoT/IIoT Applications

### DISCUSSION THREAD

**Discussion by halflauren** [OpenDDS/OpenDDS#3685] [Announcements]
Join us on Aug 25, 2022, 10:00 AM CT, for this free webinar where you'll expand your skills and learn how to build more powerful IoT and IIoT applications! 

Details:

The RtpsRelay is a horizontally scalable service that forwards RTPS messages between OpenDDS participants. OpenDDS participants without direct network connectivity can interact by forwarding messages to a mutually reachable RtpsRelay cluster.

The addition of the RtpsRelay (and other features like DDS Security and XTypes) means that OpenDDS middleware can be used for IoT and IIoT applications.

In this webinar Justin Wilson takes an in-depth look at the design and operation of the RtpsRelay.

The following topics are covered in this webinar:
- The motivation for the RtpsRelay
- RtpsRelay design
- How the RtpsRelay forwards messages
- Configuring the RtpsRelay
- Monitoring and logging
- Configuring OpenDDS applications to use the RtpsRelay
- Gotchas when operating over the public internet

[Register today!](https://us02web.zoom.us/webinar/register/9716595432101/WN_Es166sb1QgK9R_AGlroICg)

**Comment by mitza-oci** [comment]
The webinar slides and recording are available at https://objectcomputing.com/products/opendds/resources/rtpsrelay-iot
